bin/cqlsh.py

```diff
@@ -110,19 +110,8 @@ def find_zip(libprefix):
     ver = os.path.splitext(os.path.basename(cql_zip))[0][len(CQL_LIB_PREFIX):]
     sys.path.insert(0, os.path.join(cql_zip, 'cassandra-driver-' + ver))
 
-third_parties = ('futures-', 'six-')
```

Comment:
I would keep "futures-" here with the related code below. This patch should be only about six removal and nothing else.

Then there will be other ticket, which will be solving "futures" issue and there we remove it completely for trunk but for 4.0, since we have not removed six from there, we remove futures only.